### PR TITLE
Added and applied sortpom plugin to emissary pom

### PR DESCRIPTION
**Pull request by drewfarris**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -426,8 +426,8 @@
         <scope>test</scope>
         <exclusions>
           <exclusion>
-            <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
+            <groupId>javax.servlet</groupId>
```
Comment (ctubbsii) [inline pom.xml:430]: I'm surprised the plugin reordered these. Usually, the standard is to have groupId first.

File: pom.xml [hunk 2]
```diff
@@ -21,12 +21,12 @@
     <system>GitHub</system>
     <url>https://github.com/NationalSecurityAgency/emissary/issues</url>
   </issueManagement>
-  <ciManagement />
-  <distributionManagement />
+  <ciManagement></ciManagement>
+  <distributionManagement></distributionManagement>
```
Comment (ctubbsii) [inline pom.xml:25]: It's weird that these are expanded, since the plugin is configured to not expand them.

Comment (drewfarris) [reply pom.xml:25]: suggests that the configuration isn't taking effect perhaps? I'll need to look closely.

File: pom.xml (hunk 1 above)
Comment (drewfarris) [reply pom.xml:430]: I had assumed the plugin knew better that I did, but I agree, this seems strange.

**Comment by drewfarris** [comment]
> These changes look more right to me.

Yes, I discovered that putting the `configuration` clause within the `execution` block doesn't work.

**Pull request merged**